x/amm/types/pool.go

```diff
@@ -19,13 +19,14 @@ func DerivePoolRewardsPoolAddress(poolId uint64) sdk.AccAddress {
 
 func NewPool(id uint64, marketId uint64, denom0, denom1 string, tickSpacing uint32) Pool {
 	return Pool{
-		Id:             id,
-		MarketId:       marketId,
-		Denom0:         denom0,
-		Denom1:         denom1,
-		TickSpacing:    tickSpacing,
-		ReserveAddress: DerivePoolReserveAddress(id).String(),
-		RewardsPool:    DerivePoolRewardsPoolAddress(id).String(),
+		Id:               id,
+		MarketId:         marketId,
+		Denom0:           denom0,
+		Denom1:           denom1,
+		ReserveAddress:   DerivePoolReserveAddress(id).String(),
+		RewardsPool:      DerivePoolRewardsPoolAddress(id).String(),
+		TickSpacing:      tickSpacing,
+		MinOrderQuantity: utils.OneDec, // default is 1
```

Comment:
Is it decided that `TickSpacing` and `MinOrderQuantity` cannot be specified in the case of `MsgCreatePool` as a policy?

Reply:
the default `TickSpacing` is decided by `DefaultTickSpacing` module param. There can also be `DefaultMinOrderQuantity`, but with different exponents around, I thought it's not meaningful to add that param to the module. So I decided to use the default `MinOrderQuantity` as 1, since that was the desired behavior when I first implemented the new DEX, before applying `sdk.Dec` to quantities.

Reply:
got it, LGTM